content/200-concepts/100-components/02-prisma-client/032-composite-types.mdx

```diff
@@ -0,0 +1,530 @@
+---
```

Comment:
The current position in the navigation is a bit weird . the things after `CRUD` are also normal model methods like `CRUD` itself.

The first "different" one is `Advanced Type Safety`. So maybe put it in front of that?

Reply:
Agreed on positioning, especiallly since it's just for MongoDB right now.

Reply:
Unfortunately there's no space between 051-working-with-fields and 052-advanced-type-safety :( 

For now I've put it at 056, let me know how to rearrange the numbering system if you want it elsewhere

Reply:
It looks okay where it is now.

Reply:
Ah, ok, thanks. I think I'll move this after the release as a separate PR... it involves renumbering a few files so it's a bit messy and I'm also nervous of having to fix merge conflicts with the other PRs being merged today.